torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -243,6 +243,9 @@ class FullyShardedDataParallel(nn.Module, _FSDPState):
         the all-reduce times over the replication process group for some
         cluster setups.
 
+    .. note::
+        For advanced notes please refer to :ref:`fsdp_notes`.
```

Comment:
I would not make this a real ".. note:" as there are a lot of them already.
Also I think this is important enough content that we should move it at the top of FSDP doc if @awgu is happy with it.

Reply:
You suggested on slack to add it to a separate doc with notes. But I'm happy to move it elsewhere if you tell me where. I'm not sure where the FSDP doc is - I didn't find one.

Reply:
Ho no, I meant the link to the note here can be moved at the top of the FSDP docstring so that people will see it :D

Reply:
Thank you for clarifying, @albanD - I have just pushed that.